src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,14 @@
 package core.basesyntax;
 
+import java.util.Random;
+
 public class ColorSupplier {
+    private static final int COLOR_COUNT = Color.values().length;
```

Comment:
```suggestion
    private static final int COLORS = Color.values();
```

or your suggestion of name